docs/paper/paper.md

```diff
@@ -21,41 +21,40 @@ link-bibliography: false
 
-In addition, due to the flexibility of the package, users may use the functions and modular design of `PlasticParcels` to enhance their existing `Parcels` simulations and workflow. For example, users can use the initialisation maps, associated `ParticleSet` creation methods, and/or the custom physics kernels with their own `Parcels` simulations. Post-processing and analysis of the generated trajectory datasets is purposefully left to the user, however some tutorials are provided in the [`PlasticParcels` github repository](\url{https://github.com/OceanParcels/PlasticParcels}), along with the tutorials on the [`Parcels` github repository](\url{https://github.com/OceanParcels/parcels}). Below we provide an example of how `PlasticParcels` may be used, utilising a developed `ipynb` tutorial notebook.
+![Particle initialisations maps based on a) coastal mismanaged plastic waste emissions [@Jambeck2015], b) riverine mismanaged plastic waste emissions [@Meijer2021], c) fishing activity related plastic emissions [@Kroodsma2018], and d) a current global surface concentration estimate [@Kaandorp2023].\label{fig:initialisation_maps}](initialisation_maps.png){width=80%}
 
+In addition, due to the flexibility of the package, users may use the functions and modular design of `PlasticParcels` to enhance their existing `Parcels` simulations and workflow. For example, users can use the initialisation maps, associated `ParticleSet` creation methods, and/or the custom physics kernels with their own `Parcels` simulations. Post-processing and analysis of the generated trajectory datasets is purposefully left to the user, however some tutorials are provided in the [`PlasticParcels` github repository](\url{https://github.com/OceanParcels/PlasticParcels/docs/examples}), along with the tutorials in the [`Parcels` documentation](\url{https://docs.oceanparcels.org/en/latest/documentation.html}). Below we provide an example use case of `PlasticParcels`.
 
-![Particle initialisations maps based on a) current global surface concentrations, b) coastal plastic emissions, c) river plastic emissions, d) fishing activity plastic emissions.\label{fig:initialisation_maps}](initialisation_maps.png){width=80%}
 
-# Usage Examples
-**TODO fill this in once created**
+# Usage Example
 The `PlasticParcels` github repository provides several useful tutorials. Here, we briefly demonstrate how `PlasticParcels` can be used for a microplastic dispersal simulation in the Mediterranean Sea. The tutorial can be found on the `PlasticParcels` github repository, ...
```

Comment:
This still needs to be done?

Reply:
Yeah, I can create this very quickly if we decide to use the one particle per cell approach that is currently implemented. However, we wanted to do this with n particles per grid cell.

I've now created a script that saves the IDs of the grid cell, so that we can randomly distribute particles across a grid cell. The non-trivial part is how to deal with the curvilinear grid.

I think we can distribute on a [0,1]x[0,1] square, and rotate it like we do with the whole Jacobian approach. I just need to figure out how to do that rotation. Then I can run the example and update the plots.

What do you think?

Reply:
I've now included a usage example in the docs/paper/paper.md file